src/scripts.js

```diff
@@ -1,7 +1,5 @@
 import './styles.css';
 import apiCalls from './apiCalls';
-
-
-
+import './images/cookin_pan_icon.png';
```

Comment:
Good work actually solving this! To make this explicit in case future us need to come back and solve this problem again, the rules of webpack as we had it set up required an "image" folder rather than an "assets" folder